### ISSUE TITLE
pipeloader-node version mismatch

### ISSUE THREAD

**Issue by zhourunfaLi** [TinyTerra/ComfyUI_tinyterraNodes#61]
pipeloader-node version mismatch
![11111](https://github.com/TinyTerra/ComfyUI_tinyterraNodes/assets/103025008/29b150e7-9b00-4d3a-bda6-ade6a1624b1a)

**Comment by zhourunfaLi** [comment]
“pipeloader-node version mismatch”
How to fix this error? Who can help me

**Comment by TinyTerra** [comment]
If you right click on the node there should be an option called reload Node (ttN).

It currently will break the links though, I have an update to it that's almost ready that fixes the reload logic significantly

**Comment by zhourunfaLi** [comment]
@TinyTerra Parameters are misaligned after reload

**Comment by zhourunfaLi** [comment]
![微信截图_20231020153200](https://github.com/TinyTerra/ComfyUI_tinyterraNodes/assets/103025008/4a51ab35-460b-4de5-9e88-fe88e9c4c5a6)

**Comment by TinyTerra** [comment]
Correct, I haven't pushed the updated version just yet, bit quite ready

**Comment by zhourunfaLi** [comment]
ok

**Comment by TinyTerra** [comment]
Just pushed the update in its current state, converted widgets wont convert and reconnect correctly right now

**Comment by NOeSIS-yu** [comment]
Thanks for the answer, I also encountered this problem, and I thought the environment package was not installed
